k8s/overlays/crc/patch.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: route.openshift.io/v1
```

Comment:
From a clarity perspective I would put each one of these patches in a separate file and name it after the thing you're patching, so you end up with something like:

```
patchesStrategicMerge:
  - patches/route.yml
  - patches/deploymentconfig.yml
  - patches/imagestream.yml
```